tests/test_prompt.py

```diff
@@ -0,0 +1,178 @@
+"""Tests the xonsh Prompt Formatter."""
+from __future__ import unicode_literals, print_function
+
+import os
+import sys
+import time
+import socket
+from datetime import datetime, timedelta
+from unittest import TestCase, skip, mock
+
+sys.path.insert(0, os.path.abspath('..'))  # FIXME
```

Comment:
Yeah, that was copied from one of your existing test files.  near as I could tell it was so that you could import xonsh from the correct directory if you didn't happen to run it from the toplevel checkout.